Trigger/fcl/templates.fcl

```diff
@@ -212,8 +212,7 @@ Trigger : {
 			     tprSeedDeM_trigger, tprSeedDeP_trigger, cprSeedDeM_trigger, cprSeedDeP_trigger ]
 #	    Compressionlevel: 0
 	    
-	    outputCommands: [ "keep *_*_*_*",
-			      "drop *_daq_*_*" ]
+	    outputCommands: [ "keep *_*_*_*" ]
```

Comment:
Do we want to keep all the data products by default?  That could be confusing if the trigger occurs at different points (ie prescaled random triggers won't have TimeClusters, etc).   In sim production we always start with 'drop *_*_*_*' and then add.

Reply:
The line I removed was not allowing the art::Fragment dataProducts to be stored in the output file, while this is something we want. TBH, Eric wants to use a different module that lives in artdaq ("RootDAQOutput") for handling the creation of the art files. 

I think that the discussion of "which data products to keep and which not" will be handled more in detail once we will have a more robust model of the DQM. For analyzing the Trigger performance my guess is that we might need almost all of them, otherwise we would loose track of the "objects" that fired a given trigger.